src/Core__Null.res

```diff
@@ -1,4 +1,4 @@
-type t<'a> = Js.Null.t<'a>
+@unboxed type t<'a> = Js.Null.t<'a> = Value('a) | @as(null) Null
```

Comment:
Nice. I like this better than the christmas presents :grin: Although `null` is definitely also a value, it's hard to think of a name that makes the distinction accurately.

Reply:
😀🌲

Yeah the naming wasn't easy. We chose between this and Present, and I personally think Value is slightly clearer, although I do agree with your point.